src/ui/components/LeagueTopBar.tsx

```diff
@@ -81,24 +115,20 @@ const LeagueTopBar = memo(() => {
 	if (show) {
 		// Show only the first upcoming game
 		for (const game of prevGames.current) {
-			games2.push(game);
+			games2.unshift(game);
 			if (game.teams[0].pts === undefined) {
 				break;
 			}
 		}
-
-		const start = games2.length - numberOfScoreBoxes;
-		if (start > 0) {
-			games2 = games2.slice(start);
-		}
 	}
 
 	const transition = { duration: 0.2, type: "tween" };
 
 	return (
 		<div
-			className="league-top-bar flex-shrink-0 d-flex justify-content-end overflow-hidden mt-2"
+			className="league-top-bar flex-shrink-0 d-flex overflow-auto flex-row-reverse pr-3 pb-1 mt-2"
```

Comment:
Fixes the alignment for the leageTopBar.

- Apparently [there is a bug](https://bugzilla.mozilla.org/show_bug.cgi?id=962501) where the css properties of `overflow: auto;` and `justify-content: end` combined make the scrollbar disappear.
- To fix this this I used `flex-row-reverse` to align all of the items on the right side of the page combined with the `games2.unshift(game);` to correct the rendering order of games2  get around the issue.

Reply:
`pb-1` and `pr-3` are used to give space between the scrollbar and the toggle button respectively